libs/api/domains/document-provider/src/lib/document-provider.module.ts

```diff
@@ -9,9 +9,11 @@ import {
   DOCUMENT_PROVIDER_CLIENT_CONFIG_TEST,
 } from './client/documentProviderClientConfig'
 import { DocumentProviderClientTest } from './client/documentProviderClientTest'
+//import { AdminGuard, DOCUMENT_PROVIDER_ADMINS } from './admin.guard'
```

Comment:
were you going to use DOCUMENT_PROVIDER_ADMINS below?

Reply:
fixed